chronosphere/enum/enum.go

```diff
@@ -28,13 +27,13 @@ type swaggerEnum interface {
 }
 
 // Enum defines a set of enum values, where a single logical value is
```

Comment:
remove `is` because next line starts with `has`

```suggestion
// Enum defines a set of enum values, where a single logical value
```